deploy/service.yaml

```diff
@@ -4,9 +4,12 @@ metadata:
   namespace: istio-operator
   labels:
     name: istio-operator
-  name: istio-operator-metrics
+  name: istio-operator
 spec:
   ports:
+  - port: 443
+    name: webhook
+    targetPort: 443
```

Comment:
Using `8443` will ensure the operator is capable of running without using an elevated security context.

Reply:
done